ui/src/App.js

```diff
@@ -55,6 +56,7 @@ const App = () => (
         addToPlaylistDialog: addToPlaylistDialogReducer,
         activity: activityReducer,
         settings: settingsReducer,
+        slider: sliderReducer,
```

Comment:
The reducer should have a more explicit name, like `albumGridSlider`, as we can potentially have other sliders in the future

Reply:
Thank you for the review, I will make the changes, `coverSlider` might be a good alternative.